codec_bpe/audio_to_codes.py

```diff
@@ -79,29 +169,63 @@
                         channel_chunk = audio_chunk[channel]
                         if args.use_dac:
                             # prepare for model
-                            signal = AudioSignal(channel_chunk, sample_rate=sr).to(device)
-                            inputs = model.preprocess(signal.audio_data, signal.sample_rate)
+                            signal = AudioSignal(channel_chunk, sample_rate=sr).to(
+                                device
+                            )
+                            inputs = model.preprocess(
+                                signal.audio_data, signal.sample_rate
+                            )
                             # encode
                             with torch.no_grad():
-                                _, encoded_chunk, _, _, _ = model.encode(inputs, n_quantizers=args.n_quantizers)
+                                _, encoded_chunk, _, _, _ = model.encode(
+                                    inputs, n_quantizers=args.n_quantizers
+                                )
                         elif args.use_mimi:
                             # prepare for model
-                            inputs = processor(raw_audio=channel_chunk, sampling_rate=sr, return_tensors="pt").to(device)
+                            inputs = processor(
+                                raw_audio=channel_chunk,
+                                sampling_rate=sr,
+                                return_tensors="pt",
+                            ).to(device)
                             # encode
                             with torch.no_grad():
-                                encoded_chunk = model.encode(**inputs, num_quantizers=args.n_quantizers).audio_codes
+                                encoded_chunk = model.encode(
+                                    **inputs, num_quantizers=args.n_quantizers
+                                ).audio_codes
+                        elif args.use_funcodec:
+                            with torch.no_grad():
+                                encoded_chunk, _, _, _ = model(
```

Comment:
FunCodec allows the target bitrate to be set using the `bit_width: int = None` parameter. This is sort of like the `bandwidth` parameter for Encodec or the `n_quantizers` parameter for DAC: the model internally uses it to determine the number of quantizers to use. 

Please add a new optional integer command line argument to support `bit_width`. The valid ranges for this parameter are listed in the table at [https://github.com/modelscope/FunCodec?tab=readme-ov-file#available-models](https://github.com/modelscope/FunCodec?tab=readme-ov-file#available-models) under the `Bitrate` column.